Easier worklfow with panel

schroef opened the pull request:
As noted before this adds an easier workflow. It adds a panel in the Item Panel (N). Its basically sort of a copy of how "Edit Linked Libraries" works. So opening the panel we "Edit Collection", works same as before. When we enter the collection, we see a button "Return to Scene", this does as described but also automatically deletes the tmp scene. 

If there nested collection, upon selecting those, we see again "Edit Collection". When entered, we now see buttons depending on what is selected. Most times it will show "Previous Collection", which will guide to back to the prior collection we came from. It will also show "Return to Scene". When this is clicked all tmp scenes will be deleted.

This edit also will center view and zoom to item. I saw quite some folks getting zoomed out a lot but also way of track of the view

Note; I still need to clean the file. There is old and pasted coded in there i need to get rid of.

schroef committed:
## [0.2.3] - 2024-02-28
### Added
- When going into nested collection, show return Main Scene button (also removes all tmp scenes)

## [0.2.2] - 2024-02-22
### Added
- Center new created scene on enter (fixes zoom level)
- Center on instance when return to scene (fixes zoom level)
- Panel in N so we see a button (easier naviation)
- Scene will be deleted when using the new operator (button)

schroef committed:
adjsut naming convention

schroef committed:
Cleaning of comments code

schroef committed:
Added preview movie in wiki

schroef committed:
correct path video file

schroef committed:
Fixed path video preview

schroef committed:
fix path preview to gif

schroef committed:
updated tag path changelog.md

schroef committed:
updated tag link changelog.md

SuperFLEB commented:
Just wanted to let you know: I'm really glad to see your interest in this, and I'm definitely going to look at merging this in. I've just been down sick this past week and distracted by some obligations elsewhere on top of that, so I've been lagging behind. It'll probably be a bit longer before I can jump in and look it over.

schroef commented:
Take your time, it is there and won't go away

schroef committed:
Add view_layer to mix. IF we have scene and use view_layers, now it also toggles correct view_layer back on when returning to main scene

schroef committed:
Fixed previous scene issue

## [0.2.6] - 2025-04-02
### Fixed
- Fixed issue with viewlayer and multiple scene. When going back to previews scene, it would pick wrong scene from list > EIC_OP_ReturntoScene

## [0.2.5] - 2025-03-31
### Added
- Active world shader > use same as active scene. Good for matching materials using same lighting

schroef committed:
Fix typos

schroef commented:
@SuperFLEB 

I did some minor fixes and added a new world
I think its useful also have the active world from the scene where a user comes from. I found my self in issues trying to adjust materials, but i couldnt easily get the world the same. So i added "Active" which copies the world from the scene you come from
